plugin/src/main/java/spp/jetbrains/sourcemarker/VariableParser.java

```diff
@@ -0,0 +1,81 @@
+package spp.jetbrains.sourcemarker;
+
+import com.intellij.openapi.util.Pair;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static spp.jetbrains.sourcemarker.Constants.EMPTY;
+import static spp.jetbrains.sourcemarker.Constants.PATTERN_CURLY_BRACES;
+
+public class VariableParser {
+
+    private Pattern variablePattern;
+    private Pattern templateVariablePattern;
+
+    public void createPattern(List<String> scopeVars) {
+
```

Comment:
Unnecessary spacing. Here and other places

Reply:
resolved